source/plugin/configuration/defaults.rst

```diff
@@ -0,0 +1,49 @@
+=========================
+Packaging Default Configs
+=========================
+
+Example: Loading a default config from the plugin jar file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: java
+
+    import java.net.URL;
+
+    URL jarConfigFile = this.getClass().getResource("defaultConfig.conf");
+    ConfigurationLoader<CommentedConfigurationNode> loader =
+      HoconConfigurationLoader.builder().setURL(jarConfigFile).build();
+
+For this example it is important to note that the ``getResource(...)`` method works relative to the location of the
+class it is called on. So if in the above example the class lies in the package ``me.username.myplugin``, the
+``defaultConfig.conf`` file must not lie in the jar file root, but instead in the directory ``me/username/myplugin``.
+This is usually accomplished by placing the ``defaultConfig.conf`` in a resources file in your project, be careful when
```

Comment:
I suggest you break this into 2 sentences. Eg ..."your project. Be careful when"...
In general, please try not to let sentences get too long.